Utilities/Misc.md

```diff
@@ -73,9 +83,11 @@ description: Tools and Apps to make your life easier.
 
 [**Direct Download from Google Play**](https://greasyfork.org/en/scripts/33005-direct-download-from-google-play/) - Adds APK-DL, APKPure, APKCombo, APKPremier, APKMirror and Evozi download buttons to Google Play Store when browsing apps, allowing the download of the APK.
 
+[**Microsoft Store Direct Download**](https://greasyfork.org/en/scripts/394420-microsoft-store-direct-download) - Adds DDL links to Microsoft store web pages.
```

Comment:
i couldnt get it to work, can you confirm this work?

Reply:
yea, doesnt work. ms store got redesigned apparently